code/modules/mob/living/carbon/xenomorph/castes/defiler/abilities_defiler.dm

```diff
@@ -177,7 +244,27 @@
 		T.visible_message(span_danger("Noxious smoke billows from the hulking xenomorph!"))
 		count = max(0,count - 1)
 		sleep(DEFILER_GAS_DELAY)
+	toggle_particles(TRUE)
 
+// Toggles particles on or off, depending on the defined var.
+/datum/action/xeno_action/emit_neurogas/proc/toggle_particles(deactivate)
+	var/mob/living/carbon/xenomorph/X = owner
```

Comment:
please invert these (ie deactivate becomes activate)
Usually toggles mean TRUE = ON, FALSE = OFF and this is inverted